core/src/mindustry/ui/dialogs/ModsDialog.java

```diff
@@ -133,11 +144,118 @@ void setup(){
+                                        if(response.getStatus() != HttpStatus.OK){
+                                            ui.showErrorMessage(Core.bundle.format("connectfail", response.getStatus()));
+                                        }else{
+                                            Json json = new Json();
+                                            Seq<ModListing> listings = json.fromJson(Seq.class, ModListing.class, response.getResultAsString());
+                                            for(ModListing mod : listings){
+                                                if((!searchtxt.isEmpty() && !mod.repo.contains(searchtxt))||searchtxt==null) continue;
+                                                tablebrow.button(btn -> {
+                                                    btn.top().left();
+                                                    btn.margin(12f);
+                                                    btn.table(con -> {
+                                                        con.left();
+                                                        con.add("[lightgray]Name:[] " + mod.name + "\n[lightgray]Author:[] " + mod.author + "\n[accent]\uE809 " + mod.stars).width(388f).wrap().growX().pad(0f,6f,0f,6f);
+                                                        con.add().growX().pad(0f,6f,0f,6f);
+                                                    }).fillY().growX().pad(0f,6f,0f,6f);
+                                                }, Styles.clearPartialt, () -> {
+                                                    BaseDialog modSelected = new BaseDialog((Core.bundle.get("mods.browser.selected"))+ ": "+ mod.name);
+                                                    modSelected.cont.add(mod.description).width(mobile ? 400f : 500f).wrap().pad(4f).get().setAlignment(Align.center, Align.center);
+                                                    modSelected.buttons.defaults().size(200f, 54f).pad(2f);
+                                                    modSelected.setFillParent(false);
+                                                    modSelected.buttons.button("$back", Icon.left, () -> {
+                                                        modSelected.clear();
+                                                        modSelected.hide();
+                                                    });
+                                                    modSelected.buttons.button("$mods.browser.add", Icon.download, () -> {
+                                                        modSelected.hide();
+                                                        installMod[0] = () -> {
+                                                            ui.loadfrag.show();
+                                                            Core.net.httpGet("http://api.github.com/repos/" + mod.repo + "/zipball/master", loc -> {
```

Comment:
https

Reply:
h

Reply:
anuke doesnt do that

Reply:
they actually do in most places 
![image](https://user-images.githubusercontent.com/495512/101247164-93e18680-370f-11eb-8f0e-ab8e952050bc.png)